### PR TITLE
Longitude and Latitude add to location form/view

### PR DESCRIPTION
**Pull request by JosephWetherlyAshcroft**

### CODE REVIEW

**Comment by JosephWetherlyAshcroft** [comment]
@Kenneth-Siu 
@MMAcode

**Comment by Kenneth-Siu** [comment]
Can you add your migration please? :)

File: AgentDiscoveries-Backend/src/main/resources/db/migration/V8__add_longitude_latitude_to_locations.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ALTER TABLE loactions ADD latitude DECIMAL(10,10) NOT NULL DEFAULT FALSE;
+ALTER TABLE loactions ADD longitude DECIMAL(10,10) NOT NULL DEFAULT FALSE;
```
Comment (Kenneth-Siu) [inline AgentDiscoveries-Backend/src/main/resources/db/migration/V8__add_longitude_latitude_to_locations.sql:2]: Did you choose these to be of the DECIMAL type specifically? What are the advantages and disadvantages compared to, say, a FLOAT or DOUBLE?

Comment (JosephWetherlyAshcroft) [reply AgentDiscoveries-Backend/src/main/resources/db/migration/V8__add_longitude_latitude_to_locations.sql:2]: not really, but i'm going to change the numbers in brackets so the number entered cant be outrageously large

**Comment by Kenneth-Siu** [comment]
Nice, looks good

**Pull request merged**